docs/source/student_tutorial/index.rst

```diff
@@ -0,0 +1,48 @@
+============
+Getting Started
+============
+
+Revolve2 is a library that allows you to use evolutionary computing for modular robot evolution and simulation of those robots.
```

Comment:
Not needed to tell the students that but as said above it's mean to provide several things separately. but this is indeed what revolve provides for students